src/main/resources/db/migration/V1_5__referral_further_information.sql

```diff
@@ -0,0 +1,9 @@
+alter table referral
+    add column further_information text,
+    add column accessibility_needs text,
+    add column additional_needs_information text,
+    add column needs_interpreter boolean,
+    add column interpreter_language text,
```

Comment:
I seem to recall that there was a recommendation to represent the `needs_interpreter` and `interpreter_language` as a single field, something similar to `required_language`.

For languages, could we use the two-letter [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) codes? (Assuming at this point we don't care about distinctions between English dialects)

Reply:
Bringing conversation here from Slack. We need to know:

- whether the "is an interpreter required? question has already been answered  (to drive the UI)
- whether an interpreter is required
- if and only if an interpreter is required, then what language

I don’t see how we can represent all three with a single field.

As for language codes - currently it's just a free-text input in the design; since we can't ask the probation practitioner to provide a language code, we'd have to provide some other UX e.g. a dropdown for this.